x/tx/signing/get_signers.go

```diff
@@ -17,15 +19,23 @@ import (
 // message where signers are specified by the cosmos.msg.v1.signer protobuf
 // option.
 type GetSignersContext struct {
```

Comment:
I wonder if we should give this a more generic name like maybe `SigningContext`? Because it contains address codecs and the file resolver which are useful in other parts of the signing process.